blocks/library/list/index.js

```diff
@@ -7,6 +7,34 @@ import Editable from 'components/editable';
 
 const { children, prop } = hpq;
 
+function activeEditorExecCommand( command ) {
+	return () => {
+		const ed = tinymce.activeEditor;
```

Comment:
I think we'd want to avoid using globals like this. We can't assume TinyMCE is even used for the `Editable` component here. 

---
Unfortunately, it's very hard to find an alternative here. Maybe the "indent controls" should be considered as "inline toolbars" (the same way we do for formatting and alignment controls)